src/parts/Wrapper/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { StyledWrapper } from './Wrapper.style';
```

Comment:
If we retain this component we could move it to the component library, but for now it seems appropriate to add it to the parts directory alongside the other legacy layout components.